### ISSUE TITLE
Idea for a Light Weight Edition

### ISSUE THREAD

**Issue by geoffblandrbs** [qos-ch/reload4j#19]
Would it be possible to produce a reload4j-le (light-weight) edition that removes all appenders apart from the file and console appenders?

This would be useful for reassuring security teams as whilst new appenders could be added by configuration and additional JARS/classes they don't exist "by default". Also for the resulting JAR should be smaller so slightly better for containers (containers usually only log to console anyway). 

Hopefully this can use the exact same code base but exclude classes with a new reload4j-le build file. 

This kind of ties into issue #5

**Comment by ceki** [comment]
There is balance to be had. While moving chainsaw should upset many people, moving `SMTPAppender` is a bit different.

**Comment by geoffblandrbs** [comment]
>There is balance to be had. 

I agree. That is why I suggested leaving the reloadj4 JAR as is pretty much so it will work as is for anyone that does not want change.

Meanwhile this would be a new JAR that applications can use but they need to realise that it is much more limited and does not contain anything more that the basic file and console appenders.
